recipes-core/initrdscripts/initramfs-framework/bootfs

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# Copyright (C) 2018 Apertus Solutions, LLC
+# Licensed on MIT
+
+bootfs_enabled() {
+	if [ -n "${bootparam_boot}" ]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+bootfs_run() {
+	BOOTFS_DIR="/${ROOTFS_DIR}/boot/system"
+        if [ -z "$BOOTFS_DIR" ]; then
```

Comment:
Don't you mean `-d` instead of `-z`?
I don't think that string can be empty...

Reply:
So interesting point, the rootfs checks for existence because if it was set then in init the directory is created. In this case,  BOOTFS_DIR points at a directory within ROOTFS_DIR. Probably better to switch to directory check.